sdk/cognitiveservices/azure-cognitiveservices-textanalytics/pom.xml

```diff
@@ -0,0 +1,79 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Presumably the folder name for this library would be `azure-cs-textanalytics`, to save horizontal space. @weshaggard can confirm.